packages/sdk/src/sdk/api/generated/default/apis/UsersApi.ts

```diff
@@ -1567,6 +1574,39 @@ export class UsersApi extends runtime.BaseAPI {
         return await response.value();
     }
 
+    /**
+     * @hidden
+     * Gets the email encryption key for a user by their user ID
+     * Get the email encryption key for a user
+     */
+    async getUserEmailKeyRaw(params: GetUserEmailKeyRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<EmailKeyResponse>> {
+        if (params.id === null || params.id === undefined) {
+            throw new runtime.RequiredError('id','Required parameter params.id was null or undefined when calling getUserEmailKey.');
+        }
+
+        const queryParameters: any = {};
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        const response = await this.request({
+            path: `/users/{id}/emails/key`.replace(`{${"id"}}`, encodeURIComponent(String(params.id))),
+            method: 'GET',
+            headers: headerParameters,
+            query: queryParameters,
+        }, initOverrides);
+
+        return new runtime.JSONApiResponse(response, (jsonValue) => EmailKeyResponseFromJSON(jsonValue));
+    }
+
+    /**
+     * Gets the email encryption key for a user by their user ID
+     * Get the email encryption key for a user
```

Comment:
nit: since the `description` and function summaries get collated together in the docs, might be good to leave out the summary if it doesn't add any value